force-app/main/default/classes/SOQL.cls

```diff
@@ -345,17 +395,29 @@ public with sharing class SOQL {
      */
     public static void setPrimaryContact() {
         //Use the getOpportunityDetailsForClosedWon() method to get the Opportunities
-
+        List<Opportunity> getClosedWonOpp = getOpportunityDetailsForClosedWon();
         // Create a set of Account Ids from the Opportunities
-
+        Set<Id> accountIds = new Set<Id>();
+        for (Opportunity currentOpp : getClosedWonOpp) {
+            accountIds.add(currentOpp.AccountId);
+        }
         // Query the Contacts
-
+        List<Contact> contactsList = [SELECT Id, AccountId FROM Contact WHERE AccountId IN :accountIds ORDER BY FirstName];
```

Comment:
Or could it be ordered by `LastName`? (In the test method, the outcome would appear to be the same.) Honestly, this question is poorly worded and needs to be fixed.